From 64b669f027bfa94501bdf3aff79cfec70907ebb2 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 20 Dec 2017 10:05:31 +0800
Subject: [PATCH 0702/1200] config: a8k: enable support for x3310 phy firmware
 download

In order to make X3310 PHY functional, PHY's firmware needs to be
updated. This commit patch enables support for marvell phy firmware
download in armada 8k config files.

Change-Id: Iefe81c05c796184660b4d87cdde6b3d6db685981
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/55407
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
---
 configs/mvebu_db_armada8k_defconfig   | 1 +
 configs/mvebu_mcbin-88f8040_defconfig | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/mvebu_db_armada8k_defconfig b/configs/mvebu_db_armada8k_defconfig
index d0bbc94b21..c2a5be5355 100644
--- a/configs/mvebu_db_armada8k_defconfig
+++ b/configs/mvebu_db_armada8k_defconfig
@@ -30,6 +30,7 @@ CONFIG_CMD_TFTPPUT=y
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_TIME=y
 CONFIG_CMD_MVEBU_BUBT=y
+CONFIG_CMD_MVEBU_PHY_FW_DOWNLOAD=y
 CONFIG_CMD_MVEBU_RX_TRAINING=y
 CONFIG_CMD_REGULATOR=y
 CONFIG_CMD_EXT4_WRITE=y
diff --git a/configs/mvebu_mcbin-88f8040_defconfig b/configs/mvebu_mcbin-88f8040_defconfig
index b267422532..51fa8777cf 100644
--- a/configs/mvebu_mcbin-88f8040_defconfig
+++ b/configs/mvebu_mcbin-88f8040_defconfig
@@ -30,6 +30,7 @@ CONFIG_CMD_TFTPPUT=y
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_TIME=y
 CONFIG_CMD_MVEBU_BUBT=y
+CONFIG_CMD_MVEBU_PHY_FW_DOWNLOAD=y
 CONFIG_CMD_REGULATOR=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_MAC_PARTITION=y
-- 
2.22.0

